--- wpa_supplicant-2.6/wpa_supplicant/wpa_priv.c.0011~	2017-11-28 16:13:43.859376853 +0000
+++ wpa_supplicant-2.6/wpa_supplicant/wpa_priv.c	2017-11-28 16:45:29.374955024 +0000
@@ -1187,7 +1187,7 @@
 	if (daemonize && os_daemonize(pid_file) && eloop_sock_requeue())
 		goto out;
 
-	eloop_register_signal_terminate(wpa_priv_terminate, NULL);
+        eloop_register_signal_terminate((eloop_signal_handler)wpa_priv_terminate, NULL);
 	eloop_run();
 
 	ret = 0;
