diff -Naur wpa_supplicant-1.0/wpa_supplicant/dbus/dbus_new_helpers.c wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/dbus_new_helpers.c
--- wpa_supplicant-1.0/wpa_supplicant/dbus/dbus_new_helpers.c	2012-05-09 21:56:09.000000000 +0000
+++ wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/dbus_new_helpers.c	2012-05-25 09:13:42.669775649 +0000
@@ -882,7 +882,7 @@
 	const struct wpa_dbus_property_desc *dsc;
 	int i = 0;
 
-	if (iface == NULL)
+	if (iface == NULL || path == NULL)
 		return;
 
 	dbus_connection_get_object_path_data(iface->con, path,
diff -Naur wpa_supplicant-1.0/wpa_supplicant/dbus/dbus_old.c wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/dbus_old.c
--- wpa_supplicant-1.0/wpa_supplicant/dbus/dbus_old.c	2012-05-09 21:56:09.000000000 +0000
+++ wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/dbus_old.c	2012-05-25 09:13:42.669775649 +0000
@@ -419,7 +419,7 @@
 	if (wpa_s->global == NULL)
 		return;
 	iface = wpa_s->global->dbus;
-	if (iface == NULL)
+	if (iface == NULL || wpa_s->dbus_path == NULL)
 		return;
 
 	/* Only send signal if state really changed */
