diff -Naur wpa_supplicant-1.0/wpa_supplicant/dbus/fi.epitest.hostap.WPASupplicant.service.in wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/fi.epitest.hostap.WPASupplicant.service.in
--- wpa_supplicant-1.0/wpa_supplicant/dbus/fi.epitest.hostap.WPASupplicant.service.in	2012-05-09 21:56:09.000000000 +0000
+++ wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/fi.epitest.hostap.WPASupplicant.service.in	2012-05-25 08:53:45.898525233 +0000
@@ -1,5 +1,5 @@
 [D-BUS Service]
 Name=fi.epitest.hostap.WPASupplicant
-Exec=@BINDIR@/wpa_supplicant -u
+Exec=@BINDIR@/wpa_supplicant -c /etc/wpa_supplicant.conf -u -P /var/run/wpa_supplicant.pid
 User=root
 SystemdService=wpa_supplicant.service
diff -Naur wpa_supplicant-1.0/wpa_supplicant/dbus/fi.w1.wpa_supplicant1.service.in wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/fi.w1.wpa_supplicant1.service.in
--- wpa_supplicant-1.0/wpa_supplicant/dbus/fi.w1.wpa_supplicant1.service.in	2012-05-09 21:56:09.000000000 +0000
+++ wpa_supplicant-1.0.tpg/wpa_supplicant/dbus/fi.w1.wpa_supplicant1.service.in	2012-05-25 08:54:30.318525081 +0000
@@ -1,5 +1,5 @@
 [D-BUS Service]
 Name=fi.w1.wpa_supplicant1
-Exec=@BINDIR@/wpa_supplicant -u
+Exec=@BINDIR@/wpa_supplicant -c /etc/wpa_supplicant.conf -u -P /var/run/wpa_supplicant.pid
 User=root
 SystemdService=wpa_supplicant.service
