--- wpa_supplicant-0.7.3/wpa_supplicant/wpa_priv.c.cast~	2011-12-01 05:37:40.323072169 +0100
+++ wpa_supplicant-0.7.3/wpa_supplicant/wpa_priv.c	2011-12-01 05:37:43.540083426 +0100
@@ -1030,7 +1030,7 @@ int main(int argc, char *argv[])
 	if (daemonize && os_daemonize(pid_file))
 		goto out;
 
-	eloop_register_signal_terminate(wpa_priv_terminate, NULL);
+	eloop_register_signal_terminate((eloop_signal_handler)wpa_priv_terminate, NULL);
 	eloop_run();
 
 	ret = 0;
